server/src/controllers/githubController.ts

```diff
@@ -0,0 +1,18 @@
+import { logger } from "../../lib/config";
+import { Request, Response } from "express";
+import { getUserData as getData } from "../services/userService";
+
+const getUserData = async (req: Request, res: Response) => {
+  const { userName } = req.params;
+  try {
+    const userData = await getData(userName);
+    res.status(200).send(userData.data.user);
+  } catch (e) {
+    logger.error(`Failed to get github user data with userdata ${userName}`, e);
+    res.status(404).send({});
```

Comment:
Can we send back the error with the response, maybe `.send({ error: <message> })`

Reply:
Yeah